### DISCUSSION TITLE
Issues in Intro to Github workflow

### DISCUSSION THREAD

**Discussion by mohdaffan3** [community/community#122394] [GitHub Education]
### Hi everyone,

I started Intro to Github Workflow module but I think I accidently deleted some of important thing from the assignment repo. And now while submitting that repo I am not getting any of the points correct. So I thought of deleting the repo and starting over again but I am not getting an option of deleting that repo. How can I start over again in new repo ?

**Comment by ShutterStack** [comment]
If you accidentally deleted important files from your assignment repository on GitHub and want to start over with a new repository by clicking on the new repo option and creating with a new name. And if u want to delete the current repo, Head to your repo u want to delete and click on settings icon. 
Scroll down till the bottom and u'll be able to see the opttion of deleting the repo. Type the name again of your repo as it is once prompted
Once the files are deleted you wont be able to retrieve those files back so be aware about it.

**Reply by mohdaffan3** [reply]
Yes but the issue is I am not getting an option to delete the repo, I guess I do not have permission to do that action, any other thing I can do?
